packages/peregrine/lib/talons/SearchBar/useAutocomplete.js

```diff
@@ -15,23 +15,30 @@ import debounce from 'lodash.debounce';
  * @param {Boolean} props.visible - whether to show the element
  */
 export const useAutocomplete = props => {
-    const { query, valid, visible } = props;
+    const {
+        queries: { PRODUCT_SEARCH, GET_PRODUCT_FILTERS_BY_SEARCH },
+        valid,
+        visible
+    } = props;
 
-    // prepare to run query
-    const [runQuery, queryResult] = useLazyQuery(query);
-    const { data, error, loading } = queryResult;
+    // Prepare to run the queries.
+    const [runProductSearch, productResult] = useLazyQuery(PRODUCT_SEARCH);
+    const [runFilterSearch, { data: filterResult }] = useLazyQuery(
+        GET_PRODUCT_FILTERS_BY_SEARCH
+    );
 
-    // retrieve value from another field
+    // Get the search term from the field.
     const { value } = useFieldState('search_query');
 
     // Create a debounced function so we only search some delay after the last
     // keypress.
     const debouncedRunQuery = useMemo(
         () =>
             debounce(inputText => {
-                runQuery({ variables: { inputText, filters: {} } });
+                runProductSearch({ variables: { inputText, filters: {} } });
+                runFilterSearch({ variables: { search: inputText } });
```

Comment:
If we care we can merge these two back into a single query though I would write a new one and not double up on the `productSearch` query.

Reply:
Yes, I think we should merge these into one query specific to this component.

Reply:
Done!